lumicks/pylake/scan.py

```diff
@@ -245,9 +254,9 @@ def _plot(
 
         adjustment._update_limits(image_handle, image, channel)
 
-        scan_axes = self._ordered_axes()
-        axes.set_xlabel(rf"{axis_label[scan_axes[0]['axis']]} ($\mu$m)")
-        axes.set_ylabel(rf"{axis_label[scan_axes[1]['axis']]} ($\mu$m)")
+        scan_axes = self._metadata.ordered_axes
+        axes.set_xlabel(rf"{axis_label[scan_axes[0].axis]} ($\mu$m)")
+        axes.set_ylabel(rf"{axis_label[scan_axes[1].axis]} ($\mu$m)")
```

Comment:
ah yes, here is where we could use the name property for the axes, and then can get rid of `axis_label` altogether

Reply:
👍